nixos/modules/services/misc/exhibitor.nix

```diff
@@ -15,9 +15,11 @@ let
     election-port=${toString cfg.zkElectionPort}
     cleanup-period-ms=${toString cfg.zkCleanupPeriod}
     servers-spec=${concatStringsSep "," cfg.zkServersSpec}
-    auto-manage-instances=${lib.boolToString cfg.autoManageInstances}
+    auto-manage-instances=${toString cfg.autoManageInstances}
     ${cfg.extraConf}
   '';
+  # NB: toString rather than lib.boolToString on cfg.autoManageInstances is intended.
+  # Exhibitor expects it to be a 0/1 integer rather than a truthy/falsey string
```

Comment:
Given that `toString` returns `"1"` or `""`, this might strike folks as weird. I think the actual Exhibitor logic checks against `"1"` so `toString` is fine, but it might be clearer if the comment said it's okay because of that. Or just be explicit with `if foo then "1" else "0"` 😄